site/web/app/themes/sage/assets/styles/components/resource-card.sass

```diff
@@ -27,16 +27,12 @@
         -moz-text-decoration-color: #E52D27
         font-size: 14px
 
-.resources:after
-  content: ""
-  display: table
-  clear: both
+section.resources
+    display: flex
+    flex-flow: row wrap
 
-@media screen and (max-width: 770px)
-    .resource-card
-        height: 250px
-
-@media screen and (max-width: 770px)
+@media screen and (max-width: 768px)
     .resource-card
         width: 90%
```

Comment:
set the .resource-card width to 100%. allow the container of all of this to set the size / padding restrictions